checker/manual/external-tools.tex

```diff
@@ -302,17 +302,17 @@ \subsection{Explanation\label{ant-task-explanation}}
 \section{Maven\label{maven}}
```

Comment:
Thanks, Werner. The conclusion is that in Maven running on JDK 7, we can use our own javac. The additional instructions I provided in the comments on this pull request enable this.